src/vs/workbench/contrib/terminalContrib/links/browser/terminalLinkOpeners.ts

```diff
@@ -108,7 +108,14 @@ export class TerminalSearchLinkOpener implements ITerminalLinkOpener {
 		// - Grep output: <link>:<result line>
 		// This only happens when the colon is _not_ followed by a forward- or back-slash as that
 		// would break absolute Windows paths (eg. `C:/Users/...`).
-		text = text.replace(/:[^\\/][^\d]+$/, '');
+		text = text.replace(/:[^\\/\d][^\d]+$/, '');
```

Comment:
Thanks a bunch for the investigation!

I think this should actually use `*` instead of `+` since it's meant to catch one or more but `+` would mean 2 or more?

```suggestion
		text = text.replace(/:[^\\/\d][^\d]*$/, '');
```

If my suspicion is right a test for this would be great

Reply:
Hi Tyriar,

Yes I believe you're correct, an asterisk would be more appropriate and does catch the edge case I tried yesterday where `: ` was not matched, which I guess you could possibly get from a grep output!